ssh_key_rotator/custom_keygen.py

```diff
@@ -1,17 +1,17 @@
 "Module for generating public/private SSH key pairs"
 import os
 import shutil
-from typing import Tuple
 from Crypto.PublicKey import RSA
 
 
 PRIVATE_KEY_NAME: str = "key"
 PUBLIC_KEY_NAME: str = f"{PRIVATE_KEY_NAME}-cert.pub"
+KEY_SIZE_BITS = 2048
 
 
```

Comment:
in general you should be using an enveloping class - for instance,  a `KeyGenerator` - than you can put all of your functions and constants inside